tests/terraform/checks/resource/gcp/test_GoogleCloudMySqlLocalInfileOff.py

```diff
@@ -0,0 +1,173 @@
+import unittest
+
+import hcl2
+
+from checkov.terraform.checks.resource.gcp.GoogleCloudMySqlLocalInfileOff import check
+from checkov.common.models.enums import CheckResult
+
+
+class TestCloudMySqlLocalInfileOff(unittest.TestCase):
+
+    def test_failure(self):
+        hcl_res = hcl2.loads("""
+                resource "google_sql_database_instance" "tfer--gilad-002D-mysql81" {
```

Comment:
better remove all of the 'gilad' instances and replace it with something more generic, since it's a public code